Tasks/PowerShellV2/powershell.ts

```diff
@@ -42,6 +51,22 @@ async function run() {
             }
 
             input_arguments = tl.getInput('arguments') || '';
+
+            if (input_arguments) {
+                const [argsArray, telemetry] = parsePowerShellArguments(input_arguments)
```

Comment:
>parsePowerShellArguments [](http://example.com/codeflow?start=47&length=24)

Same feedback from bash task:

1. if not enableTelemetry or enableSecureArgs, we shouldn't call parsePowerShellArguments
2. if enableTelemetry and not enableSecureArgs , we should trap errors from parsePowerShellArguments and record telemetry if error occurs
3. if enableTelemetry and enableSecureArgs , let errors fall through from parsePowerShellArguments

Reply:
Fixed, thanks!
Note: we're not throwing any exceptions from the args parser, only stops next parsing in case of blockers

Reply:
yes but exceptions may fall through due to bugs e.g. index out of range, null references, etc.  it should not break due to bugs when only enableTelemetry.  (however, when enableSecureArgs it should break due to bugs)

Reply:
done